### PR TITLE
Fix .webp imports by expanding webpack loader config

### LINKED ISSUE #27411: 🐛 Bug Report: missing webpack loader for webp imports
### 📜 Description

Webpack configuration is missing a loader for handling imports of *.webp images.

### 👍 Expected behavior

It should be possible to import webp images from typescript srouce files.

### 👎 Actual Behavior with Screenshots

Webpack error is thrown.

Screenshot 2024-10-31 at 14-56-17 Backstage Example App

### 👟 Reproduction steps

Create a .webp image and try importing it into a source file.

### 📃 Provide the context for the Bug.

_No response_

### 🖥️ Your Environment

Current master branch of the backstage repo.

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by Antolius**
## Hey, I just made a Pull Request!

fixes #27411

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by Antolius** [comment]
I haven't found tests for the bundler in the cli package. I tried this out locally by adding a webp image and importing it from home page of the example-app. If you think this could be useful I can commit that change as well.

**Review by Rugvip — COMMENTED** [review]
Thank you! 👍 

This'll need to be fixed in the build configuration too, here: https://github.com/backstage/backstage/blob/ea1663376eb45753c0de4722a0e25e2faefbd82f/packages/cli/src/lib/builder/config.ts#L165-L174

**Comment by Antolius** [comment]
Tnx for the quick review! I [added support to the builder config](https://github.com/backstage/backstage/pull/27412/commits/141c237216774548f917fca0bf6b6ca0ba013a9c) as well.

File: .changeset/wise-avocados-behave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
+---
+
+Fix .webp import support by adding it to webpack loader config.
```
Comment (Rugvip) [inline .changeset/wise-avocados-behave.md:5]:
```suggestion
Added support for `.webp` files in the frontend tooling.
```

Just switching it up to be a bit more broad

**Review by Rugvip — APPROVED** [review]
Thank you! 👍 

Let's :shipit:

**Pull request merged**